src/main/java/org/tron/common/overlay/discover/node/NodeManager.java

```diff
@@ -36,19 +37,23 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
-import org.tron.common.overlay.discover.NodeHandler.State;
-import org.tron.common.overlay.discover.message.FindNodeMessage;
-import org.tron.common.overlay.discover.message.Message;
-import org.tron.common.overlay.discover.message.NeighborsMessage;
-import org.tron.common.overlay.discover.message.PingMessage;
-import org.tron.common.overlay.discover.message.PongMessage;
+import org.tron.common.net.udp.handler.EventHandler;
+import org.tron.common.net.udp.handler.MessageHandler;
+import org.tron.common.net.udp.message.Message;
+import org.tron.common.net.udp.message.discover.FindNodeMessage;
+import org.tron.common.net.udp.message.discover.NeighborsMessage;
+import org.tron.common.net.udp.message.discover.PingMessage;
+import org.tron.common.net.udp.message.discover.PongMessage;
+import org.tron.common.overlay.discover.DiscoverListener;
+import org.tron.common.net.udp.handler.UdpEvent;
+import org.tron.common.overlay.discover.node.NodeHandler.State;
 import org.tron.common.overlay.discover.table.NodeTable;
 import org.tron.common.utils.CollectionUtils;
 import org.tron.core.config.args.Args;
 import org.tron.core.db.Manager;
 
 @Component
-public class NodeManager {
+public class NodeManager implements EventHandler {
 
   static final org.slf4j.Logger logger = LoggerFactory.getLogger("NodeManager");
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Explicitly declare the visibility for "logger". ([squid:S2039](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2039))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=951&issues=AWPTJmwzA0k93xG_IQRb&open=AWPTJmwzA0k93xG_IQRb)

[ISSUE_KEY]: AWPTJmwzA0k93xG_IQRb